triage/labels.rst

```diff
@@ -10,405 +10,136 @@ Many labels are shared for both use cases, while some are dedicated
-| open          | Issue is not resolved.                                     |
-+---------------+------------------------------------------------------------+
-| closed        | The issue has been resolved (somehow).                     |
-+---------------+------------------------------------------------------------+
-
-Linked pull requests
---------------------
-A link might be added manually using the cog icon next to this field.
-Most commonly though, if the PR includes "Fixes #xxx" in its description,
-the link will be added automatically.
-
-Generating Special Links in a Comment
-=====================================
-Using the following abbreviations in a comment will automatically generate
-a link to relevant web pages.
-
-+-------------------------------------------------------------+-------------------------------------------------------+
-| Comment abbreviation                                        | Description                                           |
-+=============================================================+=======================================================+
-| ``#<number>``,                                              | Links to the tracker issue or PR ``<number>`` (they   |
-| ``GH-<number>``                                             | share the same sequence of integers on GitHub).       |
-+-------------------------------------------------------------+-------------------------------------------------------+
-| ``BPO-<number>``                                            | Links to the old bug tracker at bugs.python.org.      |
-+-------------------------------------------------------------+-------------------------------------------------------+
-| a 10-, 11-, 12-, or 40-digit hex ``<number>``               | Indicates a Git changeset identifier and              |
-|                                                             | generates a link to changeset ``<number>`` on GitHub. |
-+-------------------------------------------------------------+-------------------------------------------------------+
-
-.. _Doc: https://github.com/python/cpython/tree/main/Doc/
-.. _Grammar: https://github.com/python/cpython/tree/main/Grammar/
-.. _Lib: https://github.com/python/cpython/tree/main/Lib/
-.. _Lib/doctest.py: https://github.com/python/cpython/blob/main/Lib/doctest.py
-.. _Lib/test: https://github.com/python/cpython/tree/main/Lib/test/
-.. _Lib/test/regrtest.py: https://github.com/python/cpython/blob/main/Lib/test/regrtest.py
-.. _Lib/test/support: https://github.com/python/cpython/tree/main/Lib/test/support/
-.. _Lib/unittest: https://github.com/python/cpython/tree/main/Lib/unittest/
-.. _Modules: https://github.com/python/cpython/tree/main/Modules/
-.. _Objects: https://github.com/python/cpython/tree/main/Objects/
-.. _Parser: https://github.com/python/cpython/tree/main/Parser/
-.. _Python: https://github.com/python/cpython/tree/main/Python/
+The following labels only apply to PRs.  They are either set automatically
```

Comment:
```suggestion
The following labels only apply to :ref:`Pull Requests <pullrequest>`.  They are either set automatically
```

* Use the full term for clarity instead of repeating the acronym
* Link the appropriate page